backend/src/main/java/feedzupzup/backend/feedback/domain/FeedBackRepository.java

```diff
@@ -21,4 +23,16 @@ List<Feedback> findPageByOrganizationIdAndCursorIdOrderByDesc(
     );
 
     List<Feedback> findByIdIn(Collection<Long> ids);
-}
+
+    @Query("""
+            SELECT f
+            FROM Feedback f
+            WHERE f.organizationId = :organizationId
+            AND f.postedAt.postedAt >= :dateTime
+            """)
+    List<Feedback> findByOrganizationIdAndPostedAtAfter(
+            Long organizationId,
+            LocalDateTime dateTime
```

Comment:
p3)
'@Param' 으로 변수명을 명시해주면 좋을 것 같습니다!
```suggestion
            @Param("organizationId") Long organizationId,
            @Param("dateTime") LocalDateTime dateTime
```